report_github_metrics.rb

```diff
@@ -109,6 +109,7 @@ def parse_array_input(arg)
 teams = parse_array_input(ARGV[3])
 metric_prefix += "." unless metric_prefix.end_with?(".")
 datadog_client = Dogapi::Client.new(ENV['DATADOG_API_KEY'])
+dog.datadog_host = ARGV[6].strip
```

Comment:
Shouldn't this be `datadog_client = ARGV[6].strip`?

Reply:
good spot, corrected though should still be .datadog_host (https://github.com/DataDog/dogapi-rb#configure-the-datadog-api-url-)